docker-compose/oidc/README.adoc

```diff
@@ -0,0 +1,445 @@
+ifdef::env-site[]
+:latest-redpanda-version: {full-version}
+// All pages already have access to {latest-console-version} on the docs site.
+endif::[]
+
+This lab walks you through integrating Azure Entra ID (formerly Azure AD) with Redpanda and Redpanda Console for unified identity using OpenID Connect (OIDC). This integration allows you to use Azure Entra ID for authentication in Redpanda and Redpanda Console.
+
+Redpanda does not support mapping Entra ID groups or other token claims to Redpanda roles. As a result, authorization decisions (who can do what) are not made by Entra ID but must be configured directly in Redpanda using ACLs or role bindings. Azure Entra ID authenticates the user, but Redpanda enforces access.
+
+== Prerequisites
+
+- Access to https://portal.azure.com[Azure Portal^]
+- https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-create-new-tenant[Azure Entra ID tenant^]
+ifndef::env-site[]
+- link:https://docs.redpanda.com/current/get-started/rpk-install/[Install `rpk`] on your host machine.
+endif::[]
+ifdef::env-site[]
+- xref:get-started:rpk-install.adoc[Install `rpk`] on your host machine.
+endif::[]
+- https://docs.docker.com/compose/install/[Docker and Docker Compose^] installed on your host machine.
+
+== Set up Azure Entra ID
+
+This section guides you through creating an Azure Entra ID app registration and configuring it for use with Redpanda.
+You'll set up the necessary permissions, scopes, and claims to ensure proper authentication and authorization.
+
+=== Create an app registration
+
+In this section, you'll register a new application in Entra ID that Redpanda will use to authenticate.
+
+. Sign in to https://portal.azure.com[Azure Portal^].
+. Navigate to *Azure Active Directory* > *App registrations* > *New registration*.
+. Configure:
+** *Name*: `RedpandaUnifiedIdentity`
+** *Supported account types*: Choose *Accounts in this organizational directory only*
+** *Redirect URI*: `http://localhost:8080/auth/callbacks/oidc`
+. Click *Register*.
+
+See also: https://learn.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app[Register an application^].
+
+=== Set access token version to v2
+
+Redpanda and Redpanda Console require that your identity provider (IdP) issues **JWT-encoded access tokens** that can be cryptographically verified. To ensure this, the IdP must support OpenID Connect (OIDC), not just OAuth 2.0.
+
+By default, OAuth 2.0 allows for opaque tokens, which Redpanda does not support. OIDC extends OAuth 2.0 by specifying how tokens should be structured, signed, and validated using a discovery document and public keys.
+
+In Azure Entra ID, you must configure the app to issue v2.0 **JWT access tokens**:
+
+. In the App Registration page, go to *Manage* > *Manifest*.
+. Set `accessTokenAcceptedVersion` to `2`:
++
+[,json]
+----
+"accessTokenAcceptedVersion": 2
+----
```

Comment:
This seems a bit superfluous given that the same guidance appears directly above. Is it in case users do not know how to format the setting in json? If not, I'd delete.